tests/data/src/pep518-3.0/setup.py

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env python
 from setuptools import find_packages, setup
 
+import simple  # Test gh-4647 regression
```

Comment:
Could you make a separate test please?

A single test should only try to test one thing.

Reply:
This test was incorrect when it was initially written. There is no need to have a broken test and a correct test.

Reply:
Ah right. I think the comment can be changed to "ensure dependency is installed".